src/client/api/restful/get_outputs_id.c

```diff
@@ -75,6 +91,21 @@ size_t get_outputs_query_str_len(outputs_query_list_t *list) {
         query_str_len += strlen(elm->query_item->param);
         query_str_len += 2;  // For "&" params seperator and "=" params assignment
         break;
+      case QUERY_PARAM_HAS_NATIVE_TOKENS:
+        query_str_len += strlen(OUTPUTS_QUERY_HAS_NATIVE_TOKENS_KEY);
+        query_str_len += strlen(elm->query_item->param);
+        query_str_len += 2;  // For "&" params seperator and "=" params assignment
```

Comment:
maybe define a local constant inside `get_outputs_query_str_len` for this magic number?
```suggestion
        query_str_len += 2;  // For "&" params seperator and "=" params assignment
```